src/redux/reducers/auth.reducer.js

```diff
@@ -4,13 +4,17 @@ import {
   SIGNUP_SUCCESS,
   VERIFIED,
   VERIFICATION_FAILED,
+  LOG_OUT_SUCCESS,
+  LOG_OUT,
 } from '../actions/types/auth.type';
 
 const initialState = {
   loggedIn: false,
   verified: false,
   signupSuccess: false,
   verifailed: false,
+  loggedOut: false,
+  logOut: false,
```

Comment:
What is the difference between these two?

Reply:
@eliemugenzi , they are different loggedOut is being used on the local login with my other teammate while i am using logOut in to logout authenticated user session